plugin/cider.vim

```diff
@@ -145,7 +136,15 @@ function! s:clean_ns() abort
   call setpos("']", [0, line2, col2, 0])
 
   if expand('<cword>') ==? 'ns'
-    let res = fireplace#message({'op': 'clean-ns', 'path': p})[0]
+	let opts = { 'op': 'clean-ns', 'path': p }
+
+	if exists('g:refactor_nrepl_options')
+		for [opt_k, opt_v] in items(g:refactor_nrepl_options)
```

Comment:
I think you can use `extend` function here. Might not even need the `if`.

Reply:
You can probably do something like `extend(get(g:,'refactor_nrepl_options', {}), opts)`

Reply:
`extend` modifies the first given dict so you should create local opts dict so the global options dict is not accidentally modified.

```
let opts = {'op': 'clean-ns', 'path': p}
extend(opts, get(g:,'refactor_nrepl_options', {}))
```

Reply:
Ah, thank you. Of course there's an extend function! Much better :)